examples/embedded_resource_manager_ex.py

```diff
@@ -0,0 +1,20 @@
+import sys, os
+sys.path.append(os.getcwd())
+from locust_plugins.embedded_resource_manager import HttpUserWithResources
+from locust import task, events
+import time
+
+class TestUserWithResources(HttpUserWithResources):
+    bundle_resource_stats=False
+    default_resource_filter=".*[^(js)]$"
```

Comment:
maybe comment these properties out and just mention that you can set them? if you write it this way, a lot of people will assume they are mandatory.

Reply:
OK